.golangci.yml

```diff
@@ -0,0 +1,9 @@
+linters:
+  enable:
+    - revive
+
+run:
+  skip-dirs:
+    - scripts
+    - docs
+    - etc
```

Comment:
Maybe we can try with:
```
linters:
  enable:
    - revive

linters-settings:
  revive:
    rules:
      - name: blank-imports
      - name: context-as-argument
      - name: context-keys-type
      - name: dot-imports
      - name: error-return
      - name: error-strings
      - name: error-naming
      - name: exported
      - name: if-return
      - name: increment-decrement
      - name: var-naming
      - name: var-declaration
      - name: package-comments
      - name: range
      - name: receiver-naming
      - name: time-naming
      - name: unexported-return
      - name: indent-error-flow
      - name: errorf
      - name: empty-block
      - name: superfluous-else
      - name: unused-parameter
      - name: unreachable-code
      - name: redefines-builtin-id

run:
  skip-dirs:
    - scripts
    - docs
    - etc
  skip-files:
    - plugins/parsers/influx/machine.go*

issues:
  exclude:
    - don't use an underscore in package name
    - exported.*should have comment.*or be unexported
```

Reply:
It will:
- enable all suggested `revive` rules
- skip two "problematic" files: `plugins/parsers/influx/machine.go*`
- filter out underscore in package name and lack of comments on exported types/functions issues